templates/about2expire.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+$this->data['header'] = sprintf(
+    'Your password will expire in %s %s',
+    $this->data['daysleft'],
+    $this->data['dayOrDays']
+);
+$this->data['autofocus'] = 'yesbutton';
+
+$this->includeAtTemplateBase('includes/header.php');
+
+?>
+<p>
+  The password for your <?= htmlentities($this->data['accountName']); ?>
```

Comment:
In other templates/views we're using the translate function to get strings to support localization right? Can that be used here too?